adafruit_ble_cycling_speed_and_cadence.py

```diff
@@ -126,13 +131,13 @@ class CyclingSpeedAndCadenceService(Service):
         "Chain Ring",
     )
 
-    def __init__(self, service=None):
+    def __init__(self, service: Optional[Service] = None) -> None:
         super().__init__(service=service)
         # Defer creating buffer until we're definitely connected.
         self._measurement_buf = None
 
     @property
-    def measurement_values(self):
+    def measurement_values(self) -> Optional[Type[CSCMeasurementValues]]:
```

Comment:
This doesn't return `None`. It always returns a `CSCMeasurementValues`.
```suggestion
    def measurement_values(self) -> CSCMeasurementValues:
```

Reply:
Ah, I see, yes, you're right, but why `Type[CSCMeasurementValues]` instead of `CSCMeasurementValues` ?